File: Shared/Dto/R2D2C/ClientMeasurementStartedDto.cs
using System;
using System.Runtime.Serialization;

namespace Iit.Fibertest.Dto
{
    [DataContract]
    public class ClientMeasurementStartedDto
    {
        [DataMember]
        public ReturnCode ReturnCode { get; set; }
        [DataMember]
        public string ErrorMessage { get; set; }
        [DataMember]
        public RtuOccupationState RtuOccupationState { get; set; }

        [DataMember]
        public Guid ClientMeasurementId { get; set; }

        [DataMember]
        public Guid TraceId { get; set; }

        [DataMember]
        public OtauPortDto OtauPortDto { get; set; }

    }
}

File: Client/WebClient/src/app/models/dtos/meas-params/clientMeasurementStartedDto.ts
import { ReturnCode } from "../../enums/returnCode";
import { OtauPortDto } from "../../underlying/otauPortDto";

export class ClientMeasurementStartedDto{
    returnCode: ReturnCode;
    errorMessage: string;
    clientMeasurementId: string;
    traceId: string;
    otauPortDto: OtauPortDto;
}